### PR TITLE
Add baners for lists

### PR DESCRIPTION
**Pull request by FieryCod**
Since @kamil-stasiak did mistake by merging the PR I was forced to revert the previous PR. This PR is revert of revert.

### CODE REVIEW

File: store/view/categories.js [hunk 1]
```diff
@@ -13,32 +13,46 @@ const actionNames = {
...
 // Mutation types
-const types = {
+export const types = {
+  CHANGE_CURRENT_CATEGORY_CODE: 'CHANGE_CURRENT_CATEGORY_CODE',
   LOAD_CATEGORIES: 'LOAD_CATEGORIES',
   ADD_CODES: 'ADD_CODES',
 }
 
 // Mutation function
+const changeCurrCategory = storeHelper.createMutationFn(types.CHANGE_CURRENT_CATEGORY_CODE, 'currentCategoryCode')
```
Comment (k911) [inline store/view/categories.js]: `changeCurrCategory` => `changeCurrentCategory` ? :)

**Comment by k911** [comment]
PS. I'm afraid that soon we'll have store paths looking like this: `view/categories/currentCategoryCodeOnXxxPage`.
So I'd reconsider extracting context of page to something like `context/articlesList/categoryCode`.

File: store/view/categories.js (hunk 1 above)
Comment (FieryCod) [reply store/view/categories.js]: Thanks I've changed it a little bit :D

**Comment by FieryCod** [comment]
@k911 I refactored a little bit. I don't think it's going to happen. If you don't mind I would keep it like it is. Of course, if there would be such a problem then will be the first one who'll do the refactor ;P

**Comment by FieryCod** [comment]
@k911 Thank you :)

**Pull request merged**